analyses/molecular-subtyping-pathology/01-compile-subtyping-results.Rmd

```diff
@@ -290,7 +284,6 @@ mb_results_df <- bind_exp_strategies(mb_results_df) %>%
 cranio_results_df <- bind_exp_strategies(cranio_results_df) %>%
   inner_join(select(histologies_df,
                     Kids_First_Biospecimen_ID,
-                    integrated_diagnosis,
```

Comment:
Is this an instance where #877 applies? We don't need to change this now necessarily. Just wondering for future reference.

Reply:
hmm good point, I think #877 doesn't apply to this step since in molecular-subtyping-integrate if a sample under goes OpenPBTA subtyping the `integrated_diagnosis` is used as `harmonized_daiagnosis`.

Reply:
also just wanted to note here that since in pbta-histologies-base.tsv `integrated_diagnosis` == NA so this column was removed to be able to directly add the integrated_diagnosis from the subtyping results.